### PR TITLE
Fix issue #89

### PR DESCRIPTION
**Pull request by wuseal**

### CODE REVIEW

File: src/main/kotlin/wu/seal/jsontokotlin/utils/classblockparse/ClassCodeParser.kt [hunk 1]
```diff
@@ -93,7 +93,7 @@ class ClassCodeParser(private val classBlockString: String) {
     }
 
     private fun getPropertyKeyword(propertyLine: String): String {
-        val stringBeforeColon = propertyLine.substringBefore(":").trim()
+        val stringBeforeColon = propertyLine.substringBefore("//").substringBeforeLast(":").trim()
```
Comment (kezhenxu94) [inline src/main/kotlin/wu/seal/jsontokotlin/utils/classblockparse/ClassCodeParser.kt]: What if the json key contains multiple colons, for instance, "foo:bar:bazz"

Comment (wuseal) [reply src/main/kotlin/wu/seal/jsontokotlin/utils/classblockparse/ClassCodeParser.kt]: I have tested that, no error when the json key contains multiple `:`. but I found something exception that the camelcase seems doesn't work well

Comment (kezhenxu94) [reply src/main/kotlin/wu/seal/jsontokotlin/utils/classblockparse/ClassCodeParser.kt]: Well then it looks good to me

Comment (wuseal) [reply src/main/kotlin/wu/seal/jsontokotlin/utils/classblockparse/ClassCodeParser.kt]: New commit has been push, rename `stringBeforeColon`->`stringBeforeLastColonWithoutComment` to make it clear. And let the name contain `:` could be parsed into camel case

**Pull request merged**